### PR TITLE
Support Expressive 3

### PR DESCRIPTION
**Pull request by prolic**

### CODE REVIEW

**Comment by prolic** [comment]
Sorry, tests fail at the moment. You can take it either over from here or give me some hints on how to fix the tests.

File: composer.json [hunk 1]
```diff
@@ -11,20 +11,20 @@
...
     ],
     "require": {
-        "php": ">=5.6",
-        "http-interop/http-middleware": "^0.4.1",
-        "maximebf/debugbar": "^1.4",
-        "php-middleware/double-pass-compatibility": "^1.0",
+        "php": "^7.1",
+        "psr/http-server-handler": "^1.0",
+        "psr/http-server-middleware": "^1.0",
+        "maximebf/debugbar": "^1.15",
```
Comment (snapshotpl) [inline composer.json]: Are we need to update this dependecy?

**Comment by snapshotpl** [comment]
Thx @prolic !

**Comment by prolic** [comment]
:+1:

**Pull request merged**